dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/program/ProgramMessageStoreTest.java

```diff
@@ -216,26 +216,14 @@ public void setUpTest()
 
         channels.add( DeliveryChannel.SMS );
 
-        pmsgA = new ProgramMessage();
-        pmsgA.setText( text );
-        pmsgA.setSubject( subject );
-        pmsgA.setRecipients( recipientsA );
-        pmsgA.setMessageStatus( messageStatus );
-        pmsgA.setDeliveryChannels( channels );
-
-        pmsgB = new ProgramMessage();
-        pmsgB.setText( text );
-        pmsgB.setSubject( subject );
-        pmsgB.setRecipients( recipientsB );
-        pmsgB.setMessageStatus( messageStatus );
-        pmsgB.setDeliveryChannels( channels );
-
-        pmsgC = new ProgramMessage();
-        pmsgC.setText( text );
-        pmsgC.setSubject( subject );
-        pmsgC.setRecipients( recipientsC );
-        pmsgC.setMessageStatus( messageStatus );
-        pmsgC.setDeliveryChannels( channels );
+        pmsgA = ProgramMessage.builder().subject( text ).text( text ).recipients( recipientsA )
```

Comment:
You could add to the test the notificationTemplate parameter now that is available